src/language.rs

```diff
@@ -10,6 +10,7 @@ pub enum Language {
     Ocaml,
     Rust,
     Toml,
+    Bash,
```

Comment:
You can keep all these lists of languages ordered.

Reply:
Reordered in 9eb72856e5694